riscv/riscv.s

```diff
@@ -10,10 +10,11 @@
 # https://github.com/jameslzhu/riscv-card/blob/master/riscv-card.pdf
 
 ## Saving numbers to registers
-# There's no way to directly store a value in a register
+# There's no way to directly store a value in a register 
 # So we add 0 to a number, and add it to a register
 # zero is another way to reference register x0, which
 # always stays as 0
+# (🐣 COMMENT: I don't completely understand this part... are you saying that we don't record the value itself in the register but where that value is (the memory address?)? if so, where is the value? another register? i thought registers can't store numbers?) 
```

Comment:
Ah, yes this is confusing!

You can store numbers in registers. However, there's no nice command I found for just directly loading a number into a register, like

```asm
LOAD x18, 30
```

This is made up, but I'd want that to load the number 30 into register 18.

So, in RISC-V, what you do is use the `ADD` instruction, which adds two numbers together from registers and saves them to another register.

Specifically, I'm using `ADDI`. This stands for `add immediate`, which adds a number from a register to another number you provide and saves that to a register.

We have a register available in RISC-V (`x0`) that's a zero constant, which is also accessible by `zero`. If we add our number, lets say `30`, to the `zero` register, then save that to `x18`, it's effectively just loading `30` into register 18 since it's just adding `30 + 0`.

```asm
ADDI x18, zero, 30
```

Reply:
ahhh that makes a lot of sense! maybe we can move this section to the first code example so people can directly see the code example?

`# There are no commands to directly load numbers into registers
# Luckily, we have a register available in RISC-V (x0) that's a zero constant, which is also accessible by zero. If we add our number, lets say 1, to the zero register, then save that to x18, it's effectively just loading 1 into register 18 since it's just adding 1 + 0.
## Add immediate
# Save the value 1 into register 18
# x18 = 1
ADDI x18, zero, 1`

Reply:
@bellesea I think that's a really good idea! Go for it!